packages/x-components/static-docs/build-search-ui/README.md

```diff
@@ -22,13 +22,12 @@ Start building your search and discovery experience in a few simple steps:
 
 :::
 
-- **Discover more about the
-  [Interface X ecosystem](/develop-empathy-platform/build-search-ui/README.md#the-interface-x-ecosystem)**
-  and how Interface&nbsp;X works.
-- **[Integrate the Interface X Components](/develop-empathy-platform/build-search-ui/README.md#integrate-x-components)**
-  in your store web application.
-- **[Use & configure the Interface X Components](/develop-empathy-platform/build-search-ui/README.md#use-configure-the-x-components)**
-  in your project.
+- **Discover more about the [Interface X ecosystem](#the-interface-x-ecosystem)** and how
+  Interface&nbsp;X works.
+- **[Integrate the Interface X Components](#integrate-interface-x-components)** in your store web
+  application.
+- **[Use & configure the Interface X Components](#use--configure-the-interface-x-components)** in
```

Comment:
I meant that this link won't work both in GitHub and EmpathyDocs at the same time. GitHub renders the id of the header with an extra `-` that [EmpathyDocs](https://docs.empathy.co/develop-empathy-platform/build-search-ui/?utm_term=x#use-configure-the-interface-x-components) does not. I used the suggestion to highlight the difference, but I didn't intend you to commit it 😅, so roll it back if you can please.

```suggestion
- **[Use & configure the Interface X Components](#use-configure-the-interface-x-components)** in
```

I just wanted to raise awareness between this differences between GitHub and EmpathyDocs markdown renderers.